.github/workflows/build-copy-and-publish-accname.yml

```diff
@@ -0,0 +1,64 @@
+name: Build, Copy, and Publish accname
+on:
+  push:
+    branches: [main]
+    paths: ['accname/**']
```

Comment:
I think all of these paths should also include commons, correct?

Reply:
Thanks for catching this. Yes, I do need to update these trigger events to include the common path.